sdk/identity/azure-identity/src/workload_identity_credential.cpp

```diff
@@ -70,23 +61,21 @@ WorkloadIdentityCredential::WorkloadIdentityCredential(
   if (m_tokenFilePath.empty())
   {
     m_tokenFilePath = Environment::GetVariable(AzureFederatedTokenFileEnvVarName);
-    if (m_tokenFilePath.empty())
-    {
-      throw std::runtime_error(
```

Comment:
QQ: Why are these `std::runtime_error` instead of `Azure::AuthenticationException`?